Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'rails', '~> 7.0.4'
 gem 'sprockets-rails'
 
 # Use sqlite3 as the database for Active Record
+gem 'rspec'
 gem 'sqlite3', '~> 1.4'
```

Comment:
### {OPTIONAL}

- I noticed you used SQLite has your database engine. It will be great to use Postgres as it is usually the preferred database engine.

Reply:
Hello @peterrobert 
When I try my testing on my pc it is passing. I am sending screen shoot of it.
![Screenshot from 2022-10-28 04-12-22](https://user-images.githubusercontent.com/101916235/198433218-e3906c15-157d-4f55-ab4c-0e1954cffab8.png)

Reply:
@aron-helu Kindly try and clone your project from github and try running your project again. This just to confirm

Reply:
I did it is the same result. it is passing.

![Screenshot from 2022-10-28 04-34-16](https://user-images.githubusercontent.com/101916235/198443009-d6bdb97b-5f75-433b-a6f8-b3d271f1efc9.png)

Reply:
Well resolved! ✔️